opts/certs_test.go

```diff
@@ -1,59 +1,52 @@
 package opts
 
-import (
-	"errors"
-	"testing"
+// func TestDecodePem(t *testing.T) {
+// 	for _, tt := range []struct {
+// 		name    string
+// 		file    string
+// 		wantErr error
+// 	}{
+// 		{
+// 			name:    "SingleCert_not_exist",
+// 			file:    "file-not-exist.pem",
+// 			wantErr: unix.ENOENT,
+// 		},
+// 		{
+// 			name: "SingleCert_success",
+// 			file: "testdata/testcert.pem",
+// 		},
+// 	} {
+// 		t.Run(tt.name, func(t *testing.T) {
+// 			_, err := decodePem(tt.file)
+// 			if err != nil && !errors.Is(err, tt.wantErr) {
+// 				t.Errorf("decodePem(%s)= cert, %q, Want cert, %v", tt.file, err, tt.wantErr)
+// 			}
+// 		})
+// 	}
+// }
```

Comment:
Why comment out this test and not remove it?

Reply:
Done